config/application.rb

```diff
@@ -19,7 +19,7 @@ module DevelopmentApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
-    config.time_zone = "Europe/Paris"
+    config.time_zone = "Europe/Paris" unless Rails.env.test?
```

Comment:
I suppose there's a default value for time_zone when we're in test env ?

Reply:
The default value is no timezone (UTC)